### PR TITLE
Add AssertCheckErrorMixin to test django 'check'

### PR DESCRIPTION
**Pull request by kevinetienne**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,12 @@
 Changelog for incuna-test-utils
 ============================
 
+v7.0.0 (Upcoming)
+-----------------
```
Comment (KevinEtienne) [inline CHANGELOG.md]: I forgot we were using this syntax

**Comment by KevinEtienne** [comment]
@Ian-Foote @adam-thomas can you review please?

**Comment by adam-thomas** [comment]
Ohh, this is a neat idea!

**Comment by adam-thomas** [comment]
LGTM ;)

File: incuna_test_utils/utils.py [hunk 2]
```diff
@@ -37,3 +41,33 @@ def field_names(model):
...
+        errors = check_method([config])
+        self.assertEqual(errors, [], msg=msg)
+
+    def assert_check_other_config(self, check_method, msg=None):
+        """Assert `check_method` returns `[]` for other config."""
+        self.assertEqual(check_method([]), [], msg=msg)
+
+    def assert_check_registered(self, check_method, msg=None):
+        """Assert `check_method` is registered."""
+        self.assertIn(check_method, registry.registered_checks, msg=msg)
```
Comment (Ian-Foote) [inline incuna_test_utils/utils.py]: We should test this.

Comment (adam-thomas) [inline incuna_test_utils/utils.py]: > Quis custodiet ipsos custodes?

**Comment by adam-thomas** [comment]
Has this been abandoned? Can it be merged? :)

**Comment by Ian-Foote** [comment]
https://github.com/incuna/incuna-test-utils/pull/62#discussion_r64427424

**Comment by KevinEtienne** [comment]
I didn't have the time to do something about https://github.com/incuna/incuna-test-utils/pull/62#discussion_r64427424

**Pull request closed without merging**